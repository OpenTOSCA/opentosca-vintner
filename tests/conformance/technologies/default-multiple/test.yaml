error: Node "container" has multiple default technologies
