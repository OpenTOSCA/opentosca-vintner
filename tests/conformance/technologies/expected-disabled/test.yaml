error: Node "container" has no present technology
