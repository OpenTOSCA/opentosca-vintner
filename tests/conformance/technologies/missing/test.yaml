error: Container of technology "ansible@0" of node "container" does not exist
