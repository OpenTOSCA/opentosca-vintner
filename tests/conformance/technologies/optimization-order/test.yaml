description: order of technologies does not matter
