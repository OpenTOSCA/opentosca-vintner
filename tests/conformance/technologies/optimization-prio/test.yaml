description: technology rule for grandparent is not considered since technology rule for parent is stricter
