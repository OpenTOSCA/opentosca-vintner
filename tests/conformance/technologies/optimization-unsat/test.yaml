error: Node "container" has more than one technology
description: technology optimization requires technology pruning
