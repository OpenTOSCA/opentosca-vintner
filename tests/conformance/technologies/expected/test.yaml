error: Node "container" expected to have a technology
